src/Hosting/Hosting/test/Internal/HostTests.cs

```diff
@@ -731,9 +818,6 @@ public async Task HostedServiceCanInjectApplicationLifetime()
                    })
                    .Build())
             {
-                var lifetime = host.Services.GetRequiredService<IHostApplicationLifetime>();
-                lifetime.StopApplication();
```

Comment:
I’m not sure why this was there in this test. According to the name, this just checks that the lifetime can be injected into services, which will not require the application to be stopped.